src/ToggleSwitch/ToggleSwitch.tsx

```diff
@@ -127,7 +134,7 @@ const SwitchButton = styled.button<SwitchButtonProps>`
   }
 
   ${props => {
-    if (props.disabled) {
+    if (props['aria-disabled']) {
       return css`
         background-color: ${get('colors.switchTrack.disabledBg')};
         border-color: transparent;
```

Comment:
```suggestion
      return css`
        @media (forced-colors: active) {
          border-color: GrayText;
        }
        
        background-color: ${get('colors.switchTrack.disabledBg')};
        border-color: transparent;
```

Adds the disabled state to be visually present in high contrast mode.